src/test/api/karate/cards/cardsReminder.feature

```diff
@@ -32,46 +32,122 @@ Feature: CardsReminder
     ]
     """
 
-Scenario: ResetCardsReadsAndAcks
+    
+  * def entity1Array =
+      """
+      [   "ENTITY1_FR"
+      ]
+      """
+
+            * def entity2Array =
+      """
+      [   "ENTITY2_FR"
+      ]
+      """
+
+Scenario: Send card and receive reminder
+
+
+#Create new perimeter
+* callonce read('../common/createPerimeter.feature') {perimeter: '#(perimeter)', token: '#(authTokenAdmin)'}

```

Comment:
i realize that we are using every where `callonce` , but we can use `call` instead in a lot of place (callonce just cache the result , so it is interesting when getting token)